R/palaeorotate.R

```diff
@@ -419,108 +419,130 @@ palaeorotate <- function(occdf, lng = "lng", lat = "lat", age = "age",
       },
       error = function(e) {
         stop(paste("GPlates Web Service is not available.",
-      "Either the website is down or you are not connected to the internet."),
+                   "Either the website is down or you are not connected to the internet."),
              call. = FALSE)
       })
+
+    # Set-up for point method -------
+    # Define root
+    pbase <- "https://gws.gplates.org/reconstruct/reconstruct_points/"
+    # Setup query structure
+    query <- list(points = NULL, time = NULL,
+                  model = NULL, anchor_plate_id = 0,
```

Comment:
Does this anchor plate id matter? Should we document that we are using `0`?

Reply:
This basically indicates using their default reference frame (palaeomagnetic/mantle). For the models currently included it doesn't really matter. However, some of the new models being added are in a mantle reference frame (e.g. TorsvikCocks2017) by default and will need specifying as 1 to shift them to their palaeomagnetic reference frame. I don't think we need to update this now as it is the default behaviour. However, when we implement the major update coming, we will specify that all models are in their palaeomagnetic reference frame.